flag.go

```diff
@@ -111,7 +111,7 @@ func (f FeatureFlag) EvaluateExplain(user User, store FeatureStore) (*EvalResult
 	if user.Key == nil {
 		return nil, nil
 	}
-	events := make([]FeatureRequestEvent, 1)
+	events := make([]FeatureRequestEvent, 0)
```

Comment:
Oops. This was creating an array with one zero-value element and we were sending it to LD.